### DISCUSSION TITLE
npm start

### DISCUSSION THREAD

**Discussion by ItsLeon15** [npm/feedback#493] [CLI] [answered]
Hello there,

I have a suggestion running off the `npm WARN Local package.json exists, but node_modules missing, did you mean to install?` warn message. I was wondering if whenever someone tries to do `npm start` it asks you if you want to install node_modules automatically like below.

```
root@localhost:/# npm start

Local package.json exists, but node_modules is missing, did you mean to install? 

Enter Y to install or N to skip: 
```
Something like that?

Thanks

**Comment by MylesBorins** [comment, marked as answer]
Hey @ItsLeon15 thanks for the suggestion!!!

If you want to officially propose this change in behavior the best next step is to open an issue on the https://github.com/npm/rfcs repository
